src/styles/main.scss

```diff
@@ -0,0 +1,14 @@
+@import "./utils/mixins";
+@import "./utils/variables";
+@import "./watch/watch.scss";
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
better use a separate container for this. it is a good idea to position the body